react-sanfona/build.boot

```diff
@@ -56,6 +58,8 @@
                  "cljsjs/react-sanfona/production/react-sanfona.min.inc.js"})
     (sift :include #{#"^cljsjs"})
     (deps-cljs :name "cljsjs.react-sanfona"
-               :requires ["cljsjs.react"])
+               :requires ["cljsjs.react"
+                          "cljsjs.react-dom"
+                          "cljsjs.react-dom-server"])
```

Comment:
Looking at React-Sanfona sources, it doesn't use react-dom/server. Foreign-lib requires aren't optimized so adding the react-dom-server here will make it part of output if react-sanfona is used, even if react-dom-server is not used.